crates/wysiwyg/src/composer_model/menu_state.rs

```diff
@@ -214,7 +207,28 @@ where
                 ComposerAction::StrikeThrough,
                 ComposerAction::Link,
             ])
+        } else if contains_code_block(locations) {
+            disabled_actions.extend(vec![ComposerAction::InlineCode])
         }
         disabled_actions
     }
 }
+
+fn contains_inline_code(locations: &Vec<DomLocation>) -> bool {
+    locations
+        .iter()
+        .find(|l| {
+            matches!(
+                l.kind,
+                DomNodeKind::Formatting(InlineFormatType::InlineCode)
+            )
+        })
+        .is_some()
+}
+
+fn contains_code_block(locations: &Vec<DomLocation>) -> bool {
+    locations
+        .iter()
+        .find(|l| l.kind == DomNodeKind::CodeBlock)
+        .is_some()
```

Comment:
There is a `any` operator that is the equivalent of `find` + `is_some`

(we can clean this up in a separate PR because I'm sure we have done this in several other places 🙂 )